lib/rules/cli/4.1.yaml

```diff
@@ -1110,7 +1110,7 @@
   :cmd: oc adm diagnostics
   :options:
     :cluster-context: --cluster-context=<value>
-    :config: --kubeconfig=<value>
+    :config: --config=<value>
```

Comment:
this can be removed given it is already in global options

Reply:
@akostadinov Tracked in https://github.com/openshift/verification-tests/pull/708